core/server/translations/en.json

```diff
@@ -74,6 +74,10 @@
                 "versionMismatch": "Client request for {clientVersion} does not match server version {serverVersion}."
             },
             "auth": {
+                "publicAPIDisabled": {
+                    "error": "Permission error! Public API is disabled.",
+                    "context": "Please enable the deprecated v0.1 Public API or switch to using the v2 Content API. See https://docs.ghost.org/api/content/"
```

Comment:
We usually use "help" to output docs links.

Reply:
Pushed an update